core/cmd/hoverfly/main.go

```diff
@@ -68,13 +68,14 @@ var (
 	destination = flag.String("destination", ".", "destination URI to catch")
 	webserver   = flag.Bool("webserver", false, "start Hoverfly in webserver mode (simulate mode)")
 
-	addNew           = flag.Bool("add", false, "add new user '-add -username hfadmin -password hfpass'")
-	addUser          = flag.String("username", "", "username for new user")
-	addPassword      = flag.String("password", "", "password for new user")
-	addPasswordHash  = flag.String("password-hash", "", "password hash for new user instead of password")
-	isAdmin          = flag.Bool("admin", true, "supply '-admin false' to make this non admin user (defaults to 'true') ")
-	authEnabled      = flag.Bool("auth", false, "enable authentication, currently it is disabled by default")
-	disableBasicAuth = flag.Bool("disable-basic-auth", false, "disable Basic authentication, requiring all authentication to use API tokens")
+	addNew          = flag.Bool("add", false, "add new user '-add -username hfadmin -password hfpass'")
+	addUser         = flag.String("username", "", "username for new user")
+	addPassword     = flag.String("password", "", "password for new user")
+	addPasswordHash = flag.String("password-hash", "", "password hash for new user instead of password")
+	isAdmin         = flag.Bool("admin", true, "supply '-admin false' to make this non admin user (defaults to 'true') ")
+	authEnabled     = flag.Bool("auth", false, "enable authentication, currently it is disabled by default")
+
+	proxyAuthorizationHeader = flag.String("proxy-auth", "proxy-auth", "Switch the Proxy-Authorization header from proxy-auth `Proxy-Authorization` to header-auth `X-HOVERFLY-AUTHORIZATION`. Switching to header-auth will auto enable -https-only")
```

Comment:
Sorry, just noticed as well, in the description do you need to say what the possible options are?